include/partition/cell_storage.hpp

```diff
@@ -117,33 +123,27 @@ template <bool UseShareMemory> class CellStorageImpl
             std::size_t stride;
         };
 
-        std::size_t GetRow(NodeID node) const
```

Comment:
`GetOutWeight` and `GetInWeight` will return an empty range if `node`  is not a source or destination node. Row or column value in this case should be an invalid value and introduce an undefined behavior, so i moved both `find` calls and iterator checks to `GetOutWeight` and `GetInWeight`